auction-server/src/auction/service/verification.rs

```diff
@@ -613,6 +627,8 @@ impl Service<Svm> {
                         reason: msgs.join("\n"),
                     })
                 }
+                // Not important to check if bid slot is less than simulation slot if simulation is successful
```

Comment:
this comment is a bit confusing, i'm not sure what you're trying to clarify

Reply:
trying to clarify why we are not checking the bid slot against result.context.slot in all cases